src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseJoinConfirmationPageUiTest.java

```diff
@@ -112,7 +112,6 @@ private void testJoinConfirmation() throws Exception {
         confirmationPage.verifyHtml("/instructorCourseJoinConfirmationHTML.html");
         
         instructorHome = confirmationPage.clickConfirmButton();
-        instructorHome.verifyStatus("");
```

Comment:
was there a reason we had this test in the first place? It could have been there to ensure no status message is shown...